src/promptflow/promptflow/_cli/_pf_azure/_flow.py

```diff
@@ -1,50 +1,62 @@
 # ---------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
-from os import PathLike
-from pathlib import Path
-from typing import IO, AnyStr, Union
+import argparse
+from typing import Dict, List
 
-from promptflow._cli._params import logging_params
+from promptflow._cli._params import add_param_set, logging_params
 from promptflow._cli._pf_azure._utils import _get_azure_pf_client
 from promptflow._cli._utils import _set_workspace_argument_for_subparsers, activate_action
-from promptflow.azure._load_functions import load_flow
 
 
 def add_parser_flow(subparsers):
     """Add flow parser to the pf subparsers."""
     flow_parser = subparsers.add_parser(
         "flow",
         description="Manage flows for promptflow.",
-        help="pf flow",
+        help="pfazure flow",
     )
-    _set_workspace_argument_for_subparsers(flow_parser)
     flow_subparsers = flow_parser.add_subparsers()
     add_parser_flow_create(flow_subparsers)
     # add_parser_flow_get(flow_subparsers)
-    add_parser_flow_list(flow_subparsers)
+    # add_parser_flow_list(flow_subparsers)
     # add_parser_flow_delete(flow_subparsers)
-    add_parser_flow_download(flow_subparsers)
+    # add_parser_flow_download(flow_subparsers)
     flow_parser.set_defaults(action="flow")
 
 
 def add_parser_flow_create(subparsers):
     """Add flow create parser to the pf flow subparsers."""
+    epilog = """
+Use "--set" to set flow properties like:
+    name: Flow name that will be created in remote. Default to be flow folder name + timestamp if not specified.
+    type: Flow type. Default to be "standard" if not specified. Available types are: "standard", "evaluation", "chat".
+    description: Flow description. e.g. "--set description=<description>."
+    tags: Flow tags. e.g. "--set tags.key1=value1 tags.key2=value2."
+
+Examples:
+# Create a flow to azure portal with local flow folder.
+pfazure flow create --source <flow-folder-path> --set name=<flow-name> type=<flow-type>
```

Comment:
what's the param name for other commands? shall this be --flow?

Reply:
Changed it to `--flow`